src/button/Button.tsx

```diff
@@ -43,8 +44,8 @@ function ButtonComponent(
   ref: React.Ref<HTMLButtonElement>,
 ) {
   const {
-    size = "md",
-    variant = "primary",
+    size,
+    variant,
     leftIcon,
     rightIcon,
```

Comment:
Since we are here let's also change these to prefix and suffix

Reply:
Yeas.